modules/react/layout/stories/examples/Grid/GridLayoutInteractive.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import {Grid} from '@workday/canvas-kit-react/layout';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
+// eslint-disable-next-line no-duplicate-imports
+import {useState, useEffect, useRef} from 'react';
```

Comment:
```suggestion
import React, {useState, useEffect, useRef} from 'react';
```
and then delete the line at the top of the file

Reply:
Done